.github/workflows/manual.yml

```diff
@@ -0,0 +1,33 @@
+name: Manual R2 deployments
```

Comment:
What does this file do?

Reply:
Oops, pretty sure that's from a polluted main branch in my fork - I've fixed up the history to undo that.